src/telemetry.ts

```diff
@@ -102,6 +103,9 @@ export async function callWithTelemetry(
     }
   }
 
+  properties['isInternal'] =
+      InternalConfig.isInternal === true ? 'true' : 'false';

```

Comment:
why not using 

properties['isInternal'] = InternalConfig.isInternal  ?

Reply:
If isInternal is not boolean type, the result will always be true. Just want to make the comparison more robust.